src/components/ExportAccountDialog.js

```diff
@@ -13,6 +13,11 @@ import { useWallet } from '../utils/wallet';
 export default function ExportAccountDialog({ open, onClose }) {
   const wallet = useWallet();
   const [isHidden, setIsHidden] = useState(true);
+  const [isArrayFormat, setArrayFormat] = useState(false);
+
+  const keyOutput = isArrayFormat
+    ? `[${[].slice.call(wallet.provider.account.secretKey)}]`
+    : bs58.encode(wallet.provider.account.secretKey);
```

Comment:
To both line 19 and 20, is it worth adding the optional chaining operator e.g. `wallet?.provider?.account?.secretKey`? I'm new to the Solana codebase, so maybe all of these keys are guaranteed by the ts interfaces, if so then disregard this comment.

Reply:
Not sure about this.  @armaniferrante thoughts?

Reply:
I'm not the expert here, but I would be surprised if any of these fields were null.